cli_tools/common/utils/logging/service/logger.go

```diff
@@ -251,8 +246,8 @@ func (l *Logger) sendLogToServerWithRetry(logExtension *ComputeImageToolsLogExte
 	for i := 0; i < maxRetry; i++ {
 		// Before sending a new request, wait for a while if server asked to do so
 		if nextRequestWaitMillis > 0 {
-			nextRequestWaitMillis = 0
 			time.Sleep(time.Duration(nextRequestWaitMillis) * time.Millisecond)
+			nextRequestWaitMillis = 0
```

Comment:
What was this change about?

Reply:
Fixed a bug that nextRequestWaitMillis is reset before it's consumed. This bug ignored the waiting mechanism of log service.